File: Day_2/question5.c
#include<stdio.h>
int main()
{ int num ,r,sum=0;

    printf("enter the number for which you want to add the didgit: ");

    scanf("%d",&num);

    while(num > 0){
        
        r = num%10;
        sum = sum + r;

        num=num/10;
        
    }
    printf("The sum of digits in the number is: %d", sum);
    
    return 0;
}

File: Day_2/question5.py
#WAP to find sum of digits of a number.
num = int(input("Enter a number: "))

n = abs(num) #Handle negative numbers
digit_sum = 0

while n > 0: #Loop until the number becomes 0
    digit = n % 10 #Extract the last digit
    digit_sum += digit #Add it to the total sum
    n = n // 10 #Remove the last digit

print(f"The sum of the digits is: {digit_sum}")